### DISCUSSION TITLE
imagemagick image convert to svg

### DISCUSSION THREAD

**Discussion by aburayhannet** [ImageMagick/ImageMagick#5070] [Help]
I am using ImageMagick 3.5.1v but `setImageFormat("svg")` not working . can anyone give solution.
chack my links: [online-image-converter](https://aburayhan.net/online-image-converter)

give me this error:
`Fatal error: Uncaught ImagickException: delegate failed `'potrace' --svg --output '%o' '%i'' @ error/delegate.c/InvokeDelegate/`

**Comment by snibgo** [comment]
> delegate failed 'potrace'

That means IM has tried but failed to execute an external delegate called "potrace". Perhaps you haven't installed potrace.

**Reply by aburayhannet** [reply]
Can you help me one more?
how can I install `potrace` in Cpanel? because I didn't find the solution in online.

**Comment by snibgo** [comment]
> how can I install potrace in Cpanel?

I don't know.
